cmd/fleetctl/scripts_test.go

```diff
@@ -102,12 +102,13 @@ func TestRunScriptCommand(t *testing.T) {
 			scriptPath:   func() string { return writeTmpScriptContents(t, "\xff\xfa", ".sh") },
 			expectErrMsg: `Wrong data format. Only plain text allowed.`,
 		},
-		{
-			name:          "script already running",
-			scriptPath:    generateValidPath,
-			expectErrMsg:  fleet.RunScriptAlreadyRunningErrMsg,
-			expectPending: true,
-		},
+		// TODO(mna): adjust after the API changes to validations
```

Comment:
I.e. to address after the ` Run script endpoint changes` and `Run live script endpoint changes` bullets of the ticket are implemented.